test/unit/esm-querystring/input.js

```diff
@@ -0,0 +1,5 @@
+// Test that querystrings of various forms get stripped from esm imports
+
+import * as aardvark from "./animalFacts/aardvark?anteater";
```

Comment:
Lets add one more test similar to this one but with `.mjs` extensions for the file names as well as import specifiers